src/components/AppHeader/AppHeader.css

```diff
@@ -0,0 +1,53 @@
+.Header {
```

Comment:
__*Preference*__
If using CSS, and not a css-in-js library like emotion, convention for class names tends to be skewer/kebab-case (eg `.header`, `.header-logo-main`)

Reply:
Not going to worry about this unless you really care about it. I stuck with the existing class names.